stdlib/src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![warn(unused_crate_dependencies)]
 #![warn(unreachable_pub)]
 
+pub mod blackbox_fallbacks;
 pub mod fallback;
```

Comment:
`mod fallback` already contains the fallback implementations of black box function. I'm not sure why we need another module here.